src/cryptoadvance/specter/wallet.py

```diff
@@ -791,47 +791,69 @@ def get_address_info(self, address):
         except:
             return None
 
-    def get_electrum_watchonly(self):
+    def get_electrum_file(self):
+        electrum_devices = [
```

Comment:
Also some documentation here would be good as well. What is that function doing? What is it returning? Maybe you can add links to specifications of a "electrum-file"?

Reply:
I can't find such specification for the file format :(